lib/spack/spack/build_systems/python.py

```diff
@@ -192,6 +227,10 @@ def build_scripts(self, spec, prefix):
 
         self.setup_py('build_scripts', *args)
 
+    def build_scripts_args(self, spec, prefix):
+        """Arguments to pass to build_scripts."""
+        return []
```

Comment:
This little function has been missing since I first created `PythonPackage`. Oops...